packages/docs/.env.production

```diff
@@ -0,0 +1 @@
+GATSBY_CORE_PKG_VERSION=3.2.2
```

Comment:
Is the idea that in the actual production version we'd provide this via the command line instead of by this file?

Reply:
> Right now, I have environment variables for the current release version number. This points the 'download code' link to the appropriate place. I was thinking that as part of the Jenkins job, we could be provided the latest release number then update the .env.production file? If there is another way to get this info I'm open, but since the design system is not defined as a dependency for this project (instead we're using relative paths to access assets), this was my first idea
So it was either command line (which I'm not sure how Gatsby handles that) or I wasn't sure if our job could update this file based on the job's param

Reply:
I'll do some experimenting with the command line var...

Reply:
Sorry, should have read the whole PR description 🤦‍♂️

Reply:
Anything defined in this file should be definable in the shell, and Gatsby should automatically read the shell's environment variable

Reply:
🤔 Upon further reading, I don't know if Gatsby works like that. I see [this documentation](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#:~:text=Gatsby%20has%20built%2Din%20support,variables%20from%20a%20file%20named%20.) where the `.env` files are recommended.

I also tried to update the start command to define an environment variable in the terminal command like
```
CORE_PKG_VERSION=3.2.1 yarn --cwd ./packages/docs develop
```
And got undefined in a component

Reply:
This may take some experimentation